DXRMapFixups/DeusEx/Classes/DXRFixupM04.uc

```diff
@@ -244,6 +244,14 @@ function PreFirstEntryMapFixes()
     case "04_NYC_STREET":
         pg=Spawn(class'#var(prefix)PigeonGenerator',,, vectm(-1849,286,-487));//Near free clinic
         pg.MaxCount=3;
+
+        foreach AllActors(class'#var(prefix)FlagTrigger', ft) {
+            if (ft.event == 'MainGates') {
+                // Increase the radius to extend around the corner for NSFHQ starts
+                ft.SetCollisionSize(6600.0, ft.CollisionHeight);
```

Comment:
I'd probably bring this radius out a little bit more so that it is guaranteed to be hit before you have line of sight (drawn red line for rough idea - not actually accurate).  Otherwise fundamentally good.

![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/13684088/c316b7fe-21ec-461d-afd9-827dbc1b676a)